lisa/wlgen/rta.py

```diff
@@ -573,11 +581,21 @@ class Ramp(RTATask):
     :param cpus: the list of CPUs on which task can run.
                 .. note:: if not specified, it can run on all CPUs
     :type cpus: list(int)
+    
+    :param clamp_mins: when called with the index and load of a phase, 
```

Comment:
Looks like it could be fused in a single `get_clamps` parameter, returning a tuple of (min, max) so we need to introduce only one parameter. `get_` prefix shows that it's a function rather than just an iterable that is expected.

Reply:
The new commit fuses the parameters and renames it accordingly.